lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
```

Comment:
Another way to think about it: if you need a comment to clarify the meaning of a security feature's name, it needs a better name :-)